integration/src/test/resources/first-successful-test.xml

```diff
@@ -126,4 +126,16 @@
             </on-error-continue>
         </error-handler>
     </flow>
+
+    <flow name="firstSuccessfulInParallelForEach">
+        <set-payload value="#[[1, 2]]"/>
+        <parallel-foreach>
+            <first-successful>
+                <route>
```

Comment:
is a first-successful with just one route actually valid?

Reply:
add a TODO to validate this

Reply:
It is valid, probably makes no sense but there is no place in which we say or validate differently

Reply:
(including xml or docs)